src/EditorFeatures/Core/Implementation/CommentSelection/AbstractCommentUncommentService.cs

```diff
@@ -21,11 +21,8 @@ internal abstract class AbstractCommentUncommentService : ICommentUncommentServi
 
         public Document Format(Document document, IEnumerable<TextSpan> changes, CancellationToken cancellationToken)
         {
-            var snapshot = document.GetTextAsync(cancellationToken).WaitAndGetResult(cancellationToken).FindCorrespondingEditorTextSnapshot();
-
             var root = document.GetSyntaxRootAsync(cancellationToken).WaitAndGetResult(cancellationToken);
-            var formattingSpans = changes.Select(s => s.ToSnapshotSpan(snapshot))
-                                         .Select(s => CommonFormattingHelpers.GetFormattingSpan(root, s.Span.ToTextSpan()));
```

Comment:
Doesn't this undo some mapping between potentially different snapshots?

Why did I write this in the first place :-/